cmd/provision.go

```diff
@@ -15,36 +15,28 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/codeskyblue/go-sh"
+	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-// provisionCmd represents the provision command
 var provisionCmd = &cobra.Command{
 	Use:   "provision",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Provision KDK user",
+	Long: `Provision KDK user`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("provision called")
+		logger := logrus.New().WithField("command", "provision")
+
+		// TODO (rluckie): replace sh docker sdk
+		out, err := sh.Command("docker", "exec", "kdk", "/usr/local/bin/provision-user").Output()
```

Comment:
How would this work eventually on Windows systems?  sh.Command.

Reply:
I'll need to follow up on windows support.  TBH I haven't done any windows testing.